docs/data/material/components/text-fields/text-fields.md

```diff
@@ -224,20 +224,16 @@ Make sure that the input is larger than the label to display correctly.
 
 ### type="number"
 
-Inputs of type="number" have potential usability issues:
+We do not recommend using type="number" with a Text Field due to potential usability issues:
 
 - Allowing certain non-numeric characters ('e', '+', '-', '.') and silently discarding others
 - The functionality of scrolling to increment/decrement the number can cause accidental and hard-to-notice changes
 
 and more - see [this article](https://technology.blog.gov.uk/2020/02/24/why-the-gov-uk-design-system-team-changed-the-input-type-for-numbers/) by the GOV.UK Design System team for a more detailed explanation.
```

Comment:
copypasta from the Base UI Number Input page - to make the link more informative

```suggestion
- and more—see See [Why the GOV.UK Design System team changed the input type for numbers](https://technology.blog.gov.uk/2020/02/24/why-the-gov-uk-design-system-team-changed-the-input-type-for-numbers/) for a more detailed explanation of the limitations of `<input type="number">`
```